app/controllers/home_controller.rb

```diff
@@ -1,24 +1,15 @@
 class HomeController < ApplicationController
   before_action :get_coordinates, only: [:index]
+  GOTHENBURG = [57.700501, 11.975463]
```

Comment:
In what way is this code related to this feature?

Reply:
That's is my bad. Local gitflow mistake